integration-tests/integration/attendance/attendanceStatsSuspendedPage.cy.js

```diff
@@ -142,16 +142,17 @@ context('A user can view suspensions', () => {
           expect(suspensions[1].location).to.eq('WWI-1')
           expect(suspensions[1].activity).to.contain('Cleaner')
           expect(suspensions[1].attended).to.contain('Yes - acceptable absence')
-          expect(suspensions[2].name).to.contain('Two, Offender')
-          expect(suspensions[2].offenderNo).to.eq('G8975UK')
-          expect(suspensions[2].location).to.eq('WWI-2')
+          expect(suspensions[2].name).to.contain('Three, Offender')
```

Comment:
This update ended up actually fixing the ordering for some of the tables, they were meant to be descending by default